app/component/itinerary/NavigatorIntro/NavigatorIntroModal.js

```diff
@@ -0,0 +1,63 @@
+import Modal from '@hsl-fi/modal';
+import PropTypes from 'prop-types';
+import React, { useEffect, useState } from 'react';
+import { intlShape } from 'react-intl';
+import { configShape } from '../../../util/shapes';
+import NavigatorIntro from './NavigatorIntro';
+
+const NavigatorIntroModal = ({ isOpen, onPrimaryClick, onClose }, context) => {
+  const { config } = context;
+  const [logo, setLogo] = useState();
+
+  useEffect(() => {
+    if (!config.navigationLogo) {
+      return;
+    }
+
+    const loadLogo = async () => {
+      try {
+        const importedLogo = await import(
+          /* webpackChunkName: "main" */ `../../../configurations/images/${config.navigationLogo}`
+        );
+        setLogo(importedLogo.default);
+      } catch (error) {
+        // eslint-disable-next-line no-console
+        console.error('Error loading logo:', error);
+      }
+    };
+
+    loadLogo();
+  }, []);
+
+  return (
+    <Modal
+      appElement="#app"
+      isOpen={isOpen}
+      className="navigator-intro-modal"
+      overlayClassName="navigator-intro-modal-overlay"
+    >
+      <NavigatorIntro
+        logo={logo}
+        onPrimaryClick={onPrimaryClick}
+        onClose={onClose}
+      />
+    </Modal>
+  );
+};
+
+NavigatorIntroModal.propTypes = {
+  isOpen: PropTypes.bool.isRequired,
```

Comment:
isOpen prop is not needed.  Pass true to hslfi modal every time,.

Reply:
Streamlined in https://github.com/HSLdevcom/digitransit-ui/pull/5119/commits/dfda93fa56b1460bcdd1d63a6a4f388295d64f76